substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaLangReflectSubstitutions.java

```diff
@@ -44,6 +44,9 @@ final class Target_java_lang_reflect_Array {
 
     @Substitute
     private static boolean getBoolean(Object array, int index) {
+        if (array == null) {
+            throw new NullPointerException();
+        }
```

Comment:
Please use `} else if(...` to avoid an unnecessary line (applies to every method)